packages/core/src/services/agents/agentsAsTools.ts

```diff
@@ -80,12 +81,12 @@ export async function getToolDefinitionFromDocument({
 
   const toolDefinition: Tool = {
     description: description ?? 'An AI agent',
-    parameters: {
+    inputSchema: jsonSchema({
```

Comment:
Vercel changed from `parameters` to `inputSchema`, and also now they require a JSON schema